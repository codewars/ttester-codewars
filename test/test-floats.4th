\ Copyright 2023 nomennescio
<{ 0e -> 0e }>
<{ 1e 1e f+ -> 2e }>
<{ 2e 3e f* -> 6e }>
<{ 0e 1e 2e -> 0e 1e 2e }>
<{ 1e 1e f- 0e fcos 4e 2e f/ -> 0e 1e 2e }>
