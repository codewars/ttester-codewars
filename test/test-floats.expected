
<PASSED::>Test Passed

<PASSED::>Test Passed

<PASSED::>Test Passed

<PASSED::>Test Passed

<PASSED::>Test Passed

<FAILED::>Expected 1. , got 0. 
